charts/jaeger/templates/collector-svc.yaml

```
@@ -46,5 +46,8 @@ spec:
     {{- include "jaeger.selectorLabels" . | nindent 4 }}
     app.kubernetes.io/component: collector
   type: {{ .Values.collector.service.type }}
+{{- if and (eq .Values.collector.service.type "LoadBalancer") .Values.collector.service.loadBalancerIP }}
+    loadBalancerIP: {{ .Values.collector.service.loadBalancerIP }}
```

```suggestion
  loadBalancerIP: {{ .Values.collector.service.loadBalancerIP }}
```

I think indentation is wrong here. Could you please test locally)?

---

OK, I will test this out.